yarn-project/p2p/src/tx_pool/instrumentation.ts

```diff
@@ -35,39 +35,59 @@ export class TxPoolInstrumentation {
     });
   }
 
-  public recordTxSize(tx: Tx) {
-    this.txSize.record(tx.getSize());
+  public recordSize(poolObject: PoolObject) {
+    this.objectSize.record(poolObject.getSize());
   }
 
   /**
-   * Updates the metrics with the new transactions.
-   * @param txs - The transactions to record
+   * Updates the metrics with the new objects.
+   * @param txs - The objects to record
    */
-  public recordAddedTxs(status: string, count = 1) {
+  public recordAddedObjectsWithStatus(status: string, count = 1) {
```

Comment:
Bit of a nit, but I don't love needing 4 functions to add/remove with/without status. Can we just expose the raw `this.objects.add` function?